src/_scss/pages/keyword/header/header.scss

```diff
@@ -1,39 +1,46 @@
-@import "layouts/tabbedSearch/header/header";
+@import "layouts/default/stickyHeader/header";
 
-.search-header-container {
-    .search-header {
-        .search-title {
+.sticky-header__container {
+    .sticky-header {
+        .sticky-header__title {
             @include span-columns(8);
             @include media($medium-screen) {
                 @include span-columns(4);
             }
         }
-        .search-summary {
+
+        .sticky-header__summary {
             // Hide the search summary for small screens
             display: none;
+            font-size: $small-font-size;
+
             @include media($medium-screen) {
                 @include span-columns(8);
                 @include display(flex);
             }
-            font-size: $small-font-size;
-            .summary-title {
+
+            .sticky-header__summary_title {
```

Comment:
These don't seem like true modifiers. For one, you should [not use modifiers alone](https://en.bem.info/methodology/quick-start/#a-modifier-cant-be-used-alone); basically a modifier class should provide only the styling necessary to change its parameter. Use composition to apply the primary layout/styling for the block or element:

```
<div className="keyword-header__summary keyword-header__summary_type_count">
```